clients/js/src/doc.ts

```diff
@@ -1,74 +1,51 @@
 import * as fs from "fs";
-
 import yargs from "yargs";
 // Side effects are here to trigger before the afflicted libraries' on-import warnings can be emitted.
 // It is also imported so that it can side-effect without being tree-shaken.
 import "./side-effects";
-// https://github.com/yargs/yargs/blob/main/docs/advanced.md#example-command-hierarchy-using-indexmjs
-import * as aptos from "./cmds/aptos";
-import * as editVaa from "./cmds/editVaa";
-import * as evm from "./cmds/evm";
-import * as generate from "./cmds/generate";
-import * as info from "./cmds/info";
-import * as near from "./cmds/near";
-import * as parse from "./cmds/parse";
-import * as recover from "./cmds/recover";
-import * as submit from "./cmds/submit";
-import * as sui from "./cmds/sui";
-import * as verifyVaa from "./cmds/verifyVaa";
-import * as status from "./cmds/status";
-
+import { CLI_COMMAND_MODULES } from "./cmds";
 
 const MD_TAG = "<!--CLI_USAGE-->";
 
 async function getHelpText(cmd: any): Promise<string> {
   // Note that `yargs` is called as a function to produce a fresh copy.
-  // Otherwise the imported module is effectively a singleton where state from 
+  // Otherwise the imported module is effectively a singleton where state from
   // other commands is accumulated from repeat calls.
-  return await cmd.builder(yargs()).scriptName(`worm ${cmd.command}`).getHelp();
+  return await cmd
+    .builder(yargs())
+    .scriptName(`worm ${cmd.command}`)
+    .locale("en") //NOTE: 'locale' needed to override auto-detected locale from the user’s operating system
```

Comment:
why do we want to override the users setting?

Reply:
This is needed because we want the `README.md` file to always be written in english.

I noticed this while running `make docs` locally. In my particular case, my machine locale is set to spanish by default. As a result, the commands generated on the `README.md` file after running `make docs` was written in spanish.

I think you could reproduce it locally if you change your system's language preferences 👀

Reply:
I see, fair enough!